src/Microsoft.PowerShell.Commands.Management/resources/ServiceResources.resx

```diff
@@ -201,4 +201,7 @@
   <data name="ComputerAccessDenied" xml:space="preserve">
     <value>The command cannot be used to configure the service '{0}' because access to computer '{1}' is denied. Run PowerShell as admin and run your command again.</value>
   </data>
+  <data name="UnsupportedStartupType" xml:space="preserve">
+    <value>The startup type '{0}' is not supported by New-Service.</value>
```

Comment:
`by New-Service` needs to be changed since the same applies to `set-service` as well.